setup.py

```diff
@@ -25,6 +25,7 @@
             'pycroft_legacy_cache = legacy.cache_legacy:main',
             'pycroft_legacy_gerok_import = legacy_gerok.__main__',
             'pycroft_ldap_sync = ldap_sync.__main__',
+            'pycroft_handle_exceeded_traffic_limits = traffic.handle_exceeded_traffic_limits',
```

Comment:
@lukasjuhrich 
Does this automaticly commit the transaction which was started by the called function?

https://github.com/agdsn/pycroft/pull/187/files#diff-f9284d5e7e7a9b14181de8f6d0c99416R160

Reply:
I'm actually not sure the decorator works, so no Idea. Why'd you ask, did you plan on committing this feature untested? ;)